lms/static/sass/shared/_header.scss

```diff
@@ -357,7 +357,7 @@ header.global {
 // ====================
 
 // CASE: marketing/course discovery
-header.global-new {
+.header-global {
```

Comment:
Let me know if this resolves the issue on the sandbox, and then I'll go ahead and merge this.